student.rb

```diff
@@ -4,7 +4,7 @@ class Student < Person
   attr_reader :classroom
 
   def initialize(classroom, age, name = 'unknowm', parent_permission: true)
-    super(age, name, parent_permission)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
**You made a great effort by completing all requirements that given by previous reviewer!** 

However, while creating a new rental it is representing another error. 
You can watch it in this video:

https://user-images.githubusercontent.com/69011963/162575974-cff133f1-dfe0-451f-8c1a-55793712b69c.mov

- [ ] Please fix this issue as well to make your project to be approved!

Reply:
Thanks, @Rustamxon7  for mentioning this review, I just inserted a bug while correcting another one though 😄 , am on it as soon as possible